app/Models/Order.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Order extends Model
+{
+    protected $table = 'orders';
+    protected $fillable = ['id', 'customer_id', 'full_name', 'address', 'phone', 'order_note', 'status'];
+
+    public function customer()
+    {
+        return $this->belongsTo(Customer::class, 'customer_id', 'customer_id');
+    }
+
+    public function order_detail()
```

Comment:
method name must be written by camelCase, and with hasMany relationship this name must be plural